.kitchen.yml

```diff
@@ -16,6 +16,7 @@
 driver:
   name: "terraform"
   command_timeout: 1800
+  verify_version: false
```

Comment:
@morgante currently int tests across all modules will fail with the latest `0.13` [dev tools image](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/763) without this due to https://github.com/newcontext-oss/kitchen-terraform/issues/412

Reply:
I think we should probably add this line to all the modules.